### PR TITLE
Add table aws_lambda_function_metric_errors_daily. Closes #622

### LINKED ISSUE #622: Add table aws_lambda_function_metric_errors_daily
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

Can be used in aws_thrify_mod for saving cost when there are large numbers of errors

### PR DESCRIPTION
**Pull request by c0d3r-arnab**
# Example query results
<details>
  <summary>Results</summary>
  
```
> select
  name,
  timestamp,
  minimum,
  maximum,
  average,
  sample_count
from
  aws_lambda_function_metric_errors_daily
order by
  name,
  timestamp;
+--------------------------+---------------------+---------+---------+--------------------+--------------+
| name                     | timestamp           | minimum | maximum | average            | sample_count |
+--------------------------+---------------------+---------+---------+--------------------+--------------+
| arnab-hello-world-python | 2021-08-22 12:00:00 | 0       | 1       | 0.3333333333333333 | 6            |
| arnab-hello-world-python | 2021-08-23 12:00:00 | 1       | 1       | 1                  | 3            |
| func-encrypted           | 2021-06-30 12:00:00 | 0       | 0       | 0                  | 216          |
| function-test            | 2021-03-23 12:00:00 | 1       | 1       | 1                  | 1            |
| function-without-vpc     | 2021-08-22 12:00:00 | 0       | 0       | 0                  | 1            |
+--------------------------+---------------------+---------+---------+--------------------+--------------+
> select
  name,
  timestamp,
  round(minimum::numeric,2) as min_error,
  round(maximum::numeric,2) as max_error,
  round(average::numeric,2) as avg_error,
  sample_count
from
  aws_lambda_function_metric_errors_daily
where average < 1
order by
  name,
  timestamp;
+--------------------------+---------------------+-----------+-----------+-----------+--------------+
| name                     | timestamp           | min_error | max_error | avg_error | sample_count |
+--------------------------+---------------------+-----------+-----------+-----------+--------------+
| arnab-hello-world-python | 2021-08-22 12:00:00 | 0.00      | 1.00      | 0.33      | 6            |
| func-encrypted           | 2021-06-30 12:00:00 | 0.00      | 0.00      | 0.00      | 216          |
| function-without-vpc     | 2021-08-22 12:00:00 | 0.00      | 0.00      | 0.00      | 1            |
+--------------------------+---------------------+-----------+-----------+-----------+--------------+
> 

```
</details>

### CODE REVIEW

File: docs/tables/aws_lambda_function_metric_errors_daily.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Table: aws_lambda_function_metric_errors_daily
+
+Amazon CloudWatch Metrics provide data about the performance of your systems.  The `aws_lambda_function_metric_errors_daily` table provides metric statistics at 24 hour intervals for the last year.
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_errors_daily.md:3]:
```suggestion
Amazon CloudWatch Metrics provide data about the performance of your systems. The `aws_lambda_function_metric_errors_daily` table provides metric statistics at 24 hour intervals for the last year.
```

File: docs/tables/aws_lambda_function_metric_errors_daily.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (bigdatasourav) [inline docs/tables/aws_lambda_function_metric_errors_daily.md:4]:
```suggestion
```

**Review by bigdatasourav — CHANGES_REQUESTED** [review]
@c0d3r-arnab there are few small changes, please check otherwise looks good.

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**